sessionctx/variable/removed.go

```diff
@@ -22,10 +22,14 @@ package variable
 // careful not to return dummy data.
 
 var removedSysVars = map[string]string{
-	TiDBEnableAlterPlacement:       "alter placement is now always enabled",
-	TiDBEnableGlobalTemporaryTable: "temporary table support is now always enabled",
-	TiDBSlowLogMasking:             "use tidb_redact_log instead",
-	PlacementChecks:                "placement_checks is removed and use tidb_placement_mode instead",
+	TiDBEnableAlterPlacement:        "alter placement is now always enabled",
+	TiDBEnableGlobalTemporaryTable:  "temporary table support is now always enabled",
+	TiDBSlowLogMasking:              "use tidb_redact_log instead",
+	PlacementChecks:                 "placement_checks is removed and use tidb_placement_mode instead",
+	TiDBTopSQLPrecisionSeconds:      "not support change it now",
+	TiDBTopSQLReportIntervalSeconds: "not support change it now",
+	TiDBTopSQLMaxStatementCount:     "use tidb_top_sql_max_time_series_count instead",
+	TiDBTopSQLMaxCollect:            "use tidb_top_sql_max_meta_count instead",
```

Comment:
These variables are never provided to users. Maybe there is no need to notify them at all.